virtual_rainforest/models/abiotic/constants.py

```diff
@@ -12,3 +12,29 @@
 @dataclass(frozen=True)
 class AbioticConsts(ConstantsDataclass):
     """Dataclass to store all constants for the `abiotic` model."""
+
+    celsius_to_kelvin: float = 273.15
+    """Factor to convert temperature in Celsius to absolute temperature in Kelvin."""
+    standard_pressure: float = 101.3
+    """Standard atmospheric pressure, [kPa]"""
+    standard_mole: float = 44.6
+    """Moles of ideal gas in 1 m^3 air at standard atmosphere."""
+    molar_heat_capacity_air: float = 29.19
+    """Molar heat capacity of air, [J mol-1 K-1]."""
+    gravity: float = 9.81
+    """Gravitational acceleration constant, [m s-1]."""
+    stefan_boltzmann_constant: float = 5.67e-8
+    """Stefan-Boltzmann constant, [W m-2 K-4]"""
```

Comment:
These two constants (and maybe more) are defined in `scipy.constants`, so I'd use those values.

To be clear, I think it's fine to define `gravity` etc. here along with your other constants, but I would define it with `scipy.constants.g` instead of `9.81`.